commit f2a44f41607f0949a44b8399d56c6827a1a47fd5
Author: Sage Weil <sweil@redhat.com>
Date:   Tue Apr 4 16:05:23 2017 -0400

    mgr/Mgr: log something at end of handle_mgr_digest
    
    Saw a run that showed the notify_all pg_summary as the last
    message for this thread; hopefully we can see if it is
    blocked in notify_all() or not.
    
    Signed-off-by: Sage Weil <sage@redhat.com>

diff --git a/src/mgr/Mgr.cc b/src/mgr/Mgr.cc
index a3a00807031..2c095135b0c 100644
--- a/src/mgr/Mgr.cc
+++ b/src/mgr/Mgr.cc
@@ -550,7 +550,7 @@ void Mgr::handle_mgr_digest(MMgrDigest* m)
   // Hack: use this as a tick/opportunity to prompt python-land that
   // the pgmap might have changed since last time we were here.
   py_modules.notify_all("pg_summary", "");
-  
+  dout(10) << "done." << dendl;
   m->put();
 }
 
